README.md

```diff
@@ -10,103 +10,102 @@
+<!-- TODO: Insert Video here-->
 
-## Run ZITADEL anywhere
+## Get started
 
-### Self-hosted
+### ZITADEL Cloud
 
-You can run an automatically operated **ZITADEL** instance on a CNCF compliant Kubernetes cluster of your choice:
+The easiest way to get started with ZITADEL is to use our public cloud offering. [Subscribe to our newsletter](https://zitadel.com/v2) and we will be in touch with you as soon as the public release is live.
 
-* [CRD Mode on an existing k8s cluster](https://docs.zitadel.ch/docs/guides/installation/crd)
-* [GitOps Mode on an existing k8s cluster](https://docs.zitadel.ch/docs/guides/installation/gitops)
-* [GitOps Mode on VM/bare-metal](https://docs.zitadel.ch/docs/guides/installation/managed-dedicated-instance)  using [ORBOS](https://docs.zitadel.ch/docs/guides/installation/orbos)
+You can also discovery our new pay-as-you-go [pricing](https://zitadel.com/pricing/v2).
 
-### Software-as-a-Service
+### Install ZITADEL
 
-* **ZITADEL Cloud:** [**ZITADEL.ch**](https://zitadel.ch) is our shared cloud service hosted in Switzerland. [Get started](https://docs.zitadel.ch/docs/guides/basics/get-started) and try the free tier, including already unlimited users and all necessary security features.
-* **ZITADEL Dedicated:** We operate and support a dedicated instance of **ZITADEL** for you. [Get in touch!](https://zitadel.ch/contact/)
+- [We provide installation guides for multiple platforms here](https://docs.zitadel.com/docs/guides/installation)
 
-## Start using ZITADEL
+### Quickstarts - Integrate you app
 
-### Quickstarts
+- [Multiple Quickstarts can be found here](https://docs.zitadel.com/docs/quickstarts/introduction)
+- [And even more examples are located under](https://github.com/zitadel/zitadel-examples)
 
-See our [Documentation](https://docs.zitadel.ch/docs/quickstarts/introduction) to get started with ZITADEL quickly. Let us know, if you are missing a language or framework in the [Q&A](https://github.com/zitadel/zitadel/discussions/1717).
+> If you miss something please feel free to engage with us [here](https://github.com/zitadel/zitadel/discussions/1717)
+
+## Why ZITADEL
+
+- [API-first](https://docs.zitadel.com/docs/apis/introduction)
+- Strong audit trail thanks to [event sourcing](https://docs.zitadel.com/docs/concepts/eventstore)
+- [Actions](https://docs.zitadel.ch/docs/concepts/features/actions) to react on events with custom code
+- [Private labeling](https://docs.zitadel.com/docs/guides/customization/branding) for a uniform user experience
+- [cockroach database](https://www.cockroachlabs.com/) is the only dependency
+
+## Features
+
+- Single Sign On (SSO)
+- Passwordless with FIDO2 support
+- Username / Password
+- Multifactor authentication with OTP, U2F
+- [Identity Brokering](https://docs.zitadel.com/docs/guides/authentication/identity-brokering)
+- [Machine-to-machine (JWT profile)](https://docs.zitadel.com/docs/guides/authentication/serviceusers)
+- [Personal Access Tokens (PAT)]()
```

Comment:
Guide is missing https://github.com/zitadel/zitadel/issues/3678

Reply:
```suggestion
- Personal Access Tokens (PAT)
```